### PR TITLE
MTDSA 13476: TYS Updates for Delete Self-Employment Annual Submission TY (23-24)

### PR DESCRIPTION
**Pull request by sean-flynn-HMRC**

### CODE REVIEW

File: app/v1/services/DeleteAnnualSubmissionService.scala [hunk 1]
```diff
@@ -37,25 +39,44 @@ class DeleteAnnualSubmissionService @Inject() (connector: DeleteAnnualSubmission
       logContext: EndpointLogContext,
       correlationId: String): Future[ServiceOutcome[Unit]] = {
 
-    connector.deleteAnnualSubmission(request).map(_.leftMap(mapDownstreamErrors(desErrorMap)))
+    //connector.deleteAnnualSubmission(request).map(_.leftMap(mapDownstreamErrors(downstreamErrorMap)))
```
Comment (SamuelJacobsHMRC) [inline app/v1/services/DeleteAnnualSubmissionService.scala:42]: Can we remove this comment

File: test/v1/connectors/DeleteAnnualSubmissionConnectorSpec.scala [hunk 2]
```diff
@@ -28,49 +25,53 @@ import scala.concurrent.Future
...
 
   "deleteAnnualSubmission" should {
     "return a 204 with no body" when {
-      "the downstream call is successful" in new Test {
+      "the downstream call is successful when not taxYearSpecific" in new DesTest with Test {
+        def taxYear: TaxYear = TaxYear.fromMtd("2017-18")
+
+        val outcome = Right(ResponseWrapper(correlationId, ()))
+
+        willPut(
```
Comment (SamuelJacobsHMRC) [inline test/v1/connectors/DeleteAnnualSubmissionConnectorSpec.scala:55]: Can we added a negative test(s) for when errors are returned? See [here](https://github.com/hmrc/property-business-api/blob/main/test/v2/connectors/AmendUkPropertyPeriodSummaryConnectorSpec.scala)

File: app/v1/services/DeleteAnnualSubmissionService.scala (hunk 1 above)
Comment (sean-flynn-HMRC) [reply app/v1/services/DeleteAnnualSubmissionService.scala]: Remooved

File: test/v1/connectors/DeleteAnnualSubmissionConnectorSpec.scala (hunk 2 above)
Comment (sean-flynn-HMRC) [reply test/v1/connectors/DeleteAnnualSubmissionConnectorSpec.scala]: Done

**Review by SamuelJacobsHMRC — APPROVED** [review]

**Pull request merged**